ui/page/root/wallet_list.go

```diff
@@ -10,63 +11,63 @@ import (
 	"github.com/crypto-power/cryptopower/ui/load"
 	"github.com/crypto-power/cryptopower/ui/modal"
 	"github.com/crypto-power/cryptopower/ui/page/components"
+	"github.com/crypto-power/cryptopower/ui/utils"
 	"github.com/crypto-power/cryptopower/ui/values"
 	"github.com/crypto-power/cryptopower/wallet"
 )
 
 func (pg *WalletSelectorPage) initWalletSelectorOptions() {
-	pg.walletComponents = pg.Theme.NewClickableList(layout.Vertical)
+	pg.walletComponents = pg.Theme.NewWalletClickableList(layout.Vertical)
 }
 
 func (pg *WalletSelectorPage) loadWallets() {
+
 	wallets := pg.WL.AllSortedWalletList()
-	walletsList := make([]*load.WalletItem, 0, len(wallets))
+	walletsList := make(map[libutils.AssetType][]*load.WalletItem)
```

Comment:
so I see that the reason for this widget is this change yeah.. but i dont think this change is enough for a totally new widget. There should be a way to make it fit.

Reply:
what do you suggest

similar code but very different, especially when you try to merge the two, it introduces necessary complexity to other lists across the app